addons/ace_interact/stringtable.xml

```diff
@@ -23,7 +23,7 @@
             <Japanese>%1 チャンネル: %2</Japanese>
             <Czech>%1 Chn: %2</Czech>
             <Italian>%1 Chn: %2</Italian>
-            <French>%1 Canal: %2</French>
+            <French>%1 Canal : %2</French>
```

Comment:
```suggestion
            <French>%1 Canal1: %2</French>
```

Reply:
Sorry, but I don't understand your Suggested change, @TheMagnetar :) . That doesn't seem correct for me.

Reply:
Sorry. Typo while being on mobile

Reply:
No problems ;).
I'm sorry @TheMagnetar , but the french typographical rules impose a space BEFORE and after a colon.
So I prefer my version ;).